bindinate/bindinate.in

```diff
@@ -287,7 +287,7 @@ fi
 
 if [ -n "$REGENERATE" ] || [ "$REGENERATE_RAW" ]; then
     if [ -z "$REGENERATE_RAW" ]; then
-        REGENERATE_RAW="sources/$NAME-sharp-api.raw"
+        REGENERATE_RAW="$OUTDIR/sources/$NAME-sharp-api.raw"
```

Comment:
Then here if `$OUTDIR` is not set it will be `/sources/...` which is not right afaics.

Reply:
Right, I will default $OUTDIR to "."

Reply:
Reviewing the code, OUTDIR will never be empty, it's intialized already here: https://github.com/GLibSharp/bindinator/blob/master/bindinate/bindinate.in#L75